automation/upload_binary.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# ENV VARS
+VERSION=$(<VERSION)
+#ACCOUNT=
+#REPO=
+#ACCESS_TOKEN=
+#sourceBranch=
+
+# Create a release
+rm -f request.json response.json
+echo "{
+  \"tag_name\": \"$VERSION\",
+  \"target_commitish\": \"$sourceBranch\",
+  \"name\": \"$VERSION\",
+  \"body\": \"Release of version $VERSION\",
+  \"draft\": false,
+  \"prerelease\": false
+}" > request.json
+curl -d @request.json https://api.github.com/repos/$ACCOUNT/$REPO/releases?access_token=$ACCESS_TOKEN -o response.json
+# Parse response
+RELEASE_ID=$(sed -n '/id/{p;q;}' response.json)
+RELEASE_ID=${RELEASE_ID:8:-1}
+echo "RELEASE_ID=$RELEASE_ID"
+# Upload data
+# arm
+curl -H "Authorization:token $ACCESS_TOKEN" -H "Content-Type:application/gzip" --data @docker-arm-$VERSION.tar.xz https://uploads.github.com/repos/$ACCOUNT/$REPO/releases/$RELEASE_ID/assets?name=docker-arm-$VERSION.tar.xz
```

Comment:
Space missing after the colon. Also, it's a good idea to put double quotes around all the arguments that have variables in them. For example: ```"@docker-arm-$VERSION.tar.xz"```